toonz/sources/toonz/filebrowserpopup.cpp

```diff
@@ -407,7 +407,9 @@ TFilePath GenericLoadFilePopup::getPath() {
 //***********************************************************************************
 
 GenericSaveFilePopup::GenericSaveFilePopup(const QString &title)
-    : FileBrowserPopup(title) {}
+    : FileBrowserPopup(title) {
+  connect(m_nameField, SIGNAL(returnPressedNow()), this, SLOT(onOkPressed()));
```

Comment:
It is just a matter of taste, but how about to use `QAbstractButton::animateClick` slot like this ?

`connect(m_nameField, SIGNAL(returnPressedNow()), m_okButton, SLOT(animateClick()));`

Which shows the button-clicking animation before executing the saving process so that users can easily understand what process is invoked by their action.